apps/gatsby/src/AppConfig/AppConfig.js

```diff
@@ -114,22 +119,19 @@ export class AppConfig extends React.Component {
       previewUrl,
       contentSyncUrl,
       webhookUrl,
-      previewWebhookUrl,
       authToken,
     } = this.state;
 
     const validPreview = !previewUrl || isValidUrl(previewUrl);
     const validContentSync = !contentSyncUrl || isValidUrl(contentSyncUrl);
     const validWebhook = !webhookUrl || isValidUrl(webhookUrl);
-    const validPreviewWebhook = !previewWebhookUrl || isValidUrl(previewWebhookUrl);
```

Comment:
As it turns out my fix in #570 was not needed and actually never added in the first place since we didn't need the variable anywhere. What we really needed was to remove `previewWebhookUrl` altogether.